### PR TITLE
Refactor add comment

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Refactor add comment.

### Added

- Test

### Changed

- Refactor add comment and migrated to CRUD

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b refactor_add_comment -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [xx] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: trailblazer/server/api.py [hunk 1]
```diff
@@ -293,8 +293,8 @@ def put_set_analysis_uploaded():
 
 @blueprint.route("/set-analysis-status", methods=["PUT"])
 def set_analysis_status():
-    """Update analysis status of a case with supplied status."""
     put_request: Response.json = request.json
+    """Update analysis status of a case with supplied status."""
```
Comment (diitaz93) [inline trailblazer/server/api.py:297]: This docstring seems to be one line below where it should

**Review by diitaz93 — APPROVED** [review]
Looks good 👍

**Comment by henrikstranneheim** [comment]
### API ✅ 
Screenshot 2023-09-12 at 13 13 03

**Comment by henrikstranneheim** [comment]
Deployed to production:
```
Log deploy... done.
trailblazer, version 18.0.5
```

**Pull request merged**